db/migrate/20151106043618_create_facial_hair.rb

```diff
@@ -0,0 +1,12 @@
+class CreateFacialHair < ActiveRecord::Migration
+  def change
+    create_table :facial_hair do |t|
+      t.references :user
```

Comment:
The concept of ```has_one``` is always interesting.  This table schema is actually set up for ```has_many```.  ```belongs_to``` is generally preferable to actually enforce a one-to-one relationship.  Two competing users in the system can easily make two facial hair entries for the same user unless you restrict them via a unique index.  I'd suggest you either enforce this constraint "A user can only have one facial hair", or make the relationship between user and facial_hair a "belongs_to".  Or was it your intention to have many facial hair entries per user and track their history of facial hairs?